api/models/resource.util.js

```diff
@@ -40,7 +40,7 @@ exports.search = async (query, fields) => {
 };
 
 exports.create = async (params) => {
-  let resource = new Resource({});
+  let resource = new Resource(params);
   await resource.save();
   resource = exports.update(resource, params);
```

Comment:
I get a `ValidationError: desc: Path `desc` is required., name: Path `name` is required.` when it's new Resource({}) since name and desc isn't provided yet